compiler/rustc_session/src/options.rs

```diff
@@ -379,6 +379,8 @@ mod desc {
     pub const parse_passes: &str = "a space-separated list of passes, or `all`";
     pub const parse_panic_strategy: &str = "either `unwind` or `abort`";
     pub const parse_on_broken_pipe: &str = "either `kill`, `error`, or `inherit`";
+    pub const parse_patchable_function_entry: &str =
+        "total_nops and optionally prefix_nops (defaulting prefix_nops=0)";
```

Comment:
This help string doesn't mention the comma syntax at all. How about this:

> either two integers (total_nops,prefix_nops) or one integer (total_nops)

Reply:
I adjusted the string to "either two comma separated integers (total_nops,prefix_nops) or one integer (total_nops)" to make it more clear.